price/priceService.ts

```diff
@@ -1,21 +1,37 @@
-const baseUrl:string = 'https://dydx-testnet.imperator.co/v4';
+import {baseUrl} from "./url.js";
 
-function fetchCandles(pair:string,timeFrame:string):void{
-    fetch(`${baseUrl}/candles/perpetualMarkets/${pair}?resolution=${timeFrame}`)
-    .then(response => {
-        if(!response.ok){
-            throw new Error('Network response was not ok');
-        }
-        else{
-            return response.json();
+type Pair = string;
+type TimeFrame = string;
+
+
+async function fetchData(pair:Pair,timeFrame:TimeFrame): Promise<string> {
```

Comment:
I think the return type is not a string, but an array of candles.

Reply:
yes the response is object that contain all candles separate whit coma (,) and I return that object response the format is :
{
//some info
},
{
//some info
}
..